.napari/config.yml

```diff
@@ -3,13 +3,16 @@
 labels:
   ontology: EDAM-BIOIMAGING:alpha06
   terms:
-  - Medical imaging
-  - Microscopy
-  - 2D image
-  - 3D image
-  - Image time series
-  - Object detection
+  - Image segmentation
   - Cell segmentation
+  - Object detection
   - Model-based segmentation
+  - Deep learning
   - Convolutional neural network
   - Supervised learning
+  - Microscopy
+  - Medical imaging
+  - 2D image
+  - 3D image
+  - Image time series
+  - Multi-channel image
```

Comment:
Hard to see the difference when you change the order...

Reply:
Also has nothing to do with the topic of the PR.

Reply:
If you look at the [hub preview page](https://preview.napari-hub.org/stardist/stardist-napari/10), you see that `Image segmentation` is a parent of `Cell segmentation` and thus redundant. I purposely only added non-redundant topics when I initially updated the list.